datumaro/components/validator.py

```diff
@@ -57,7 +59,10 @@ def __init__(self, task_type=None, ann_type=None, far_from_mean_thr=None):
 
         self.task_type = task_type
         self.ann_type = ann_type
+
         self.far_from_mean_thr = far_from_mean_thr
+        self.imbalance_ratio_thr = self.DEFAULT_IMBALANCE_RATIO
```

Comment:
Consider putting this into the `ClassificationValidator`, if this is not supposed to be used anywhere else.

Reply:
It is commonly used by `ClassificationValidator` and `DetectionValidator`. So I think this is the right place.